com.unity.shadergraph/Editor/Data/Nodes/UV/FlipbookNode.cs

```diff
@@ -35,6 +36,19 @@ public override bool hasPreview
 
         string GetFunctionName()
         {
+            if (m_InvertX && m_InvertY)
+            {
+                return $"Unity_Flipbook_InvertXY_{concretePrecision.ToShaderString()}";
+            }
+            if (m_InvertX)
+            {
+                return $"Unity_Flipbook_InvertX_{concretePrecision.ToShaderString()}";
+            }
+            if (m_InvertY)
+            {
+                return $"Unity_Flipbook_InvertY_{concretePrecision.ToShaderString()}";
+            }
+
             return $"Unity_Flipbook_{concretePrecision.ToShaderString()}";
```

Comment:
Less code duplication and more straightforward control flow:
```suggestion
string invertText = string.Empty;
if(m_InvertX && m_InvertY)
    invertText = "InvertXY_";
else if(m_InvertX)
    invertText = "InvertX_";
else if(m_InvertY)
    invertText = InvertY_;

return $"Unity_Flipbook_{invertText}{concretePrecision.ToShaderString()}";
```

Reply:
done!